src/client/app/strings/chinese-simplified.ts

```diff
@@ -156,6 +156,10 @@ export function getChineseSimplifiedStrings(): ClientStrings {
         newSnippetDescription: '创建新代码段',
         importDescription: '创建YAML或GitHub gist的要点摘要代码段',
 
+        // Outlook only strings
+
+        noRunInOutlook: englishSubstitutesForNotYetTranslated.noRunInOutlook,
```

Comment:
Could you please file (re-open) the localization bugs, to inform our localizers about the need to incrementally translate a new string?